### PR TITLE
Add some code philosophy discussion points

### PR DESCRIPTION
**Pull request by MrPowers**

### CODE REVIEW

**Comment by robertkossendey** [comment]
@MrPowers should I resolve the conflicts?

**Comment by MrPowers** [comment]
@robertkossendey - yea, would be great if you could resolve the conflicts for this one, thank you!

**Comment by robertkossendey** [comment]
@MrPowers done 👍

File: README.md [hunk 1]
```diff
@@ -269,6 +269,50 @@ Here's the ending result:
 +----+----+----+
 ```
 
+Notice that the duplicate `col1` value was not appended.  If a normal append operation was run, then the Delta table would contain two rows of data with `col1` equal to 2.
```
Comment (brayanjuls) [inline README.md:272]: It seems that this line got duplicated at line 316.

**Pull request merged**